chapter_attention-mechanisms/attention-cues.md

```diff
@@ -200,8 +205,20 @@ we consider a simple case where
 the attention weight is one only when the query and the key are the same; otherwise it is zero.
 
 ```{.python .input}
-#@tab all
-attention_weights = d2l.eye(10).reshape(1, 1, 10, 10)
+#@tab mxnet
```

Comment:
use `#@tab mxnet, pytorch` to merge the same code block

Reply:
Okay